resources/rest-service/cloudify/types/types.yaml

```diff
@@ -1065,6 +1065,11 @@ data_types:
         type: string
         default: ""
         required: true
+      labels:
+        description: >
+          A list of single-entry dicts describing labels to apply to the blueprint.
```

Comment:
`labels` are applied to the deployments.

`blueprint_labels` apply to the blueprints themselves.